src/components/hero.js

```diff
@@ -0,0 +1,187 @@
+import { useStaticQuery } from "gatsby"
+import React from "react"
+import styled from "styled-components"
+import { graphql } from "gatsby"
+import { FaHeart, FaCalendarAlt, FaRobot, FaBell, FaLink } from "react-icons/fa"
+import colors from "../common/colors"
+import breakpoints from "../common/breakpoints"
+
+const StyledInnerWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  height: 50vh;
+  width: 100%;
+  @media (min-width: ${breakpoints.tablet}px) {
+    min-height: 70vh;
+  }
+  @media (min-width: ${breakpoints.medium}px) {
+    min-height: 85vh;
+  }
+`
+
+const HeroBackground = styled.div`
+    position: relative;
+    top: 0;
+    margin: 0 auto;
+    background-image: url('${({ image }) => image}');
+    background-size: cover;
+    filter: blur(27px);
+    min-height: 50vh;
+    z-index: -1;
+    width: 100%;
+    max-width: 1340px;
```

Comment:
Where is this coming from? Why do we want to cap the hero width?

Reply:
removed the max width